src/seahawk/seahawk_deck/pilot_input.py

```diff
@@ -77,21 +76,27 @@ def reset(self):
         self.__track_state = 0b0000
 
 
-class InputXboxOne(Node):
+class PilotInput(Node):
     """
     Class that implements the joystick input
     """
 
     def __init__(self):
         """
-        Initialize 'input_xbox_one' node
+        Initialize 'pilot_input' node
         """
-        super().__init__("input_xbox_one")
+        super().__init__("pilot_input")
 
+        # Create publishers and subscriptions
         self.subscription = self.create_subscription(Joy, "joy", self.__callback, 10)
         self.__twist_pub = self.create_publisher(Twist, "desired_twist", 10)
         self.__claw_pub = self.create_publisher(Bool, "claw_state", 10)
-        
+
+        # Create and store parameter which determines which throttle curve
+        # the pilot wants to use named 'throttle_curve_choice'. Defaults to '0'
+        self.declare_parameter("throttle_curve_choice", "0")
```

Comment:
Suggestion (strong): The parameter is created but not stored. You should create a member variable (such as `self.throttle_curve_choice`) to hold the value of the parameter so that you don't have to call `self.get_parameter("throttle_curve_choice").value` to get the value every time.

Reply:
```py
# Create and store parameter which determines which throttle curve
# the pilot wants to use named 'throttle_curve_choice'. Defaults to '0'
self.declare_parameter("throttle_curve_choice", "0")
self.__throttle_curve_choice = self.get_parameter("throttle_curve_choice").value
```
Added member variable

Reply:
This isn't set anywhere else.